content/articles/2018-07-broom-0-5-0.Rmarkdown

```diff
@@ -0,0 +1,127 @@
+---
+title: broom 0.5.0
+author: Alex Hayes
+date: "2018-07-18"
+slug: broom-0-5-0
+description: > 
+  broom 0.5.0 is on CRAN.
+categories:
+  - package
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+options(tibble.print_min = 5, tibble.print_max = 5)
+```
+
+We are delighted to announced that [broom](https://broom.tidyverse.org/) 0.5.0 is now available on CRAN. broom 0.5.0 is a major new release featuring new changes that affect both users and developers. See the [News]( https://broom.tidyverse.org/news/) for a detailed list of changes.
+
+## Tibble output
+
+All tidiers should now return `tibble`s rather than `data.frame`s. This allows broom to take advantage of the nice tibble print method and the more consistent behavior of tibbles:
+
+```{r}
+library(broom)
+
+fit <- lm(mpg ~ ., mtcars)
+tidy(fit)
+```
+
+
+These changes will mostly likely affect you when:
+
+- subset with `[`, which always returns a tibble
+- set rownames on a tibble, which is deprecated
+- use augment methods on models with matrix covariates specified in a formula, which will error
+
+Augment will error with matrix covariates because tibbles are more strict about their contents than data frames. Data frames allow users to specify columns in a matrix, like so:
+
+```{r}
+y <- 1:3
+x <- matrix(1:9, nrow = 3)
+df <- data.frame(x, y)
+```
+
+Tibbles do not:
+
+```{r, error = TRUE}
+library(tibble)
+
+tibble(x, y)
+```
+
+Modelling functions will occassional create a data frame like this, but since the model frame can't be coerced a tibble method, `augment` will fail:
+
+```{r, error = TRUE}
+fit <- lm(y ~ x, df)
+augment(fit)
+```
+
+In some cases, explicitly passing the original dataset via the `data` argument can resolve this:
+
+```{r}
+augment(fit, data = df)
+```
+
+In any case, support for matrix-columns is on the way in dplyr and in a release cycle or two this won't be an issue.
```

Comment:
Maybe kill the "In any case" (since the prior sentence begins with "In some cases…" it's ambiguous as to whether or not you're using "in any case" idiomatically or to mean "in any and all cases").